src/hotspot/cpu/x86/nativeInst_x86.cpp

```diff
@@ -386,7 +386,6 @@ void NativeJump::check_verified_entry_alignment(address entry, address verified_
 //
 void NativeJump::patch_verified_entry(address entry, address verified_entry, address dest) {
   // complete jump instruction (to be inserted) is in code_buffer;
-#ifdef _LP64
```

Comment:
Do you need this change? Meaning, does it add substantially to this fix? Looks like it does not?

I'd omit it here, so the patch is cleanly backportable. We will cleanup the remnants of x86_32 port in due course.

Reply:
I don't need it. I'll take it out before the RFR. Thanks for pointing it out!

Reply:
Should I file an RFE or do you already have everything tracked?

Reply:
There is an umbrella RFE, I created the task for `nativeInst` here: https://bugs.openjdk.org/browse/JDK-8355472 -- feel free to take it and put this hunk and other cleanups in `nativeInst_x86*` there :)